packages/vega5-extension/package.json

```diff
@@ -33,10 +33,11 @@
     "@jupyterlab/rendermime-interfaces": "^1.3.0-alpha.3",
     "@phosphor/coreutils": "^1.3.0",
     "@phosphor/widgets": "^1.6.0",
-    "vega-embed": "3.18.2"
+    "vega": "^5.3.1",
+    "vega-embed": "^4.0.0",
+    "vega-lite": "^3.0.0"
```

Comment:
@domoritz Do these need to be specified here for any reason, or are the `vega` and `vega-lite` pulled in by `vega-embed` sufficient?

Reply:
Vega-Embed now only has peer dependencies. I think it's much better for you to be able to specify the versions you want regardless of the Vega-Embed version.

Reply:
Cool, thanks.

Reply:
Discussion is here: https://github.com/vega/vega-embed/issues/136